lasagne/regularization.py

```diff
@@ -1,3 +1,53 @@
+"""
+Functions to apply regularization to the weights in a network.
+
+We provide functions to calculate the L1 and L2 penalty. Penalty functions
+take a tensor as input and calculate the penalty contribution from that tensor:
+
+.. autosummary::
+    :nosignatures:
```

Comment:
I prefer short parameter names, and I doubt anyone is going to be confused by `x`. But I don't feel that strongly about it.